qiita_pet/support_files/doc/source/admin/portals.rst

```diff
@@ -0,0 +1,72 @@
+.. _portals:
+
+.. index:: portals
+
+User Portals
+============
+
+User portals allow a user to cordon off specific studies and analyses into their
+ own user interface, which we call a portal, selectively slicing the data
+ available through that portal.
+
+The main ``QIITA`` portal gives access to all studies in the database, while
+adding studies to a specific portal and changing the config file to point to the
+portal will show the same interface, but with only the selected studies. To
+change the portal being accessed, simply change the `portal` setting in the
+qiita config file. This must correspond to a portal name in the database.
+
+Creating/Deleting a portal
+--------------------------
+Portals are created using the command line interface. A portal may be added
+using the command ``qiita-env create-portal PORTAL_NAME PORTAL_DESC``.
+Conversely, a portal may be deleted by the command
+``qiita-env remove-portal PORTAL_NAME``.
+Note, however, that you can not remove a portal that has one or more
+studies/analyses attached to it. The portal must be completely connectionless
+to be deleted.
+
+Adding/Deleting studies on a portal
+-----------------------------------
+Studies will automatically be added to the main ``QIITA`` portal, as well as
+the portal set in the config file if different than ``QIITA``. To add or remove
+a study from a portal, there are two methods: Command line and web interface.
+
+Command line additions and deletions can be done using the
+``qiita db portal [add/remove]-studies PORTAL_NAME STUDY_LIST`.
+The ``PORTAL`` is the name of the portal to add or remove from, and the
+``STUDY_LIST`` is a space separated list of study IDs to act on. For example,
+to add studies 5, 6, and 7 to the portal "EMP", you would use the following
+command:
+
+``qiita db portal add-studies EMP 5 6 7``
+
+To add from the web interface, you must have admin privileges and be on the
+``QIITA`` portal. You can then access the portals editing page through the
+admin menu dropdown. On this page, you can select all studies required,
+followed by the portal, and then add or remove as needed.
+
+Adding/Deleting analyses on a portal
+------------------------------------
+Analyses will automatically be added to the main ``QIITA`` portal, as well as
+the portal set in the config file if different than ``QIITA``. Because the
+analyses are heavily dependent on what studies and data are available, **WE
+STRONGLY ADVISE NOT CHANGING THE PORTAL ASSIGNMENTS FOR ANALYSES.**
```

Comment:
Got it.

On (Jul-06-15|12:15), Joshua Shorenstein wrote:
> > +to add studies 5, 6, and 7 to the portal "EMP", you would use the following
> > +command:
> > +
> > +``qiita db portal add-studies EMP 5 6 7``
> > +
> > +To add from the web interface, you must have admin privileges and be on the
> > +``QIITA`` portal. You can then access the portals editing page through the
> > +admin menu dropdown. On this page, you can select all studies required,
> > +followed by the portal, and then add or remove as needed.
> > +
> > +Adding/Deleting analyses on a portal
> > +------------------------------------
> > +Analyses will automatically be added to the main ``QIITA`` portal, as well as
> > +the portal set in the config file if different than ``QIITA``. Because the
> > +analyses are heavily dependent on what studies and data are available, **WE
> > +STRONGLY ADVISE NOT CHANGING THE PORTAL ASSIGNMENTS FOR ANALYSES.**
> 
> Tried it with the danger, and it looks good, but I think it may get confused with the pictures for the interface, as seen here:
> <img width="1144" alt="screen shot 2015-07-06 at 12 13 02 pm" src="https://cloud.githubusercontent.com/assets/3079066/8530893/ae800430-23d8-11e5-8bd0-411f49ad9fa9.png">
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1314/files#r33971380